src/main/java/com/softserve/teamproject/controller/UserProfileController.java

```diff
@@ -34,4 +36,15 @@ public void setUserProfileService(UserProfileService userProfileService) {
   public User getUserProfile(Principal principal) {
     return userProfileService.getUserProfile(principal.getName());
   }
+
+  /**
+   * Get groups from user location
+   * @param principal authorized user
+   * @return groups in user location
+   */
+  @RequestMapping(value = "/user/location/groups", method = RequestMethod.GET)
+  @ResponseBody
```

Comment:
Seems like we don't need responce body annotation, because:
Types that carry this annotation are treated as controllers where @RequestMapping methods assume @ResponseBody semantics by default.
(с) https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/bind/annotation/RestController.html